audit/projects/kubernetes-public/service-accounts/k8s-infra-gcp-auditor@kubernetes-public.iam.gserviceaccount.com/iam.json

```diff
@@ -2,8 +2,7 @@
   "bindings": [
     {
       "members": [
-        "serviceAccount:k8s-infra-prow-build-trusted.svc.id.goog[test-pods/k8s-infra-gcp-auditor]",
-        "serviceAccount:kubernetes-public.svc.id.goog[test-pods/k8s-infra-gcp-auditor]"
```

Comment:
Expected from https://github.com/kubernetes/k8s.io/pull/1974

This should have been removed ages ago, this was over a year ago when I was testing out a prow build cluster in the `kubernetes-public` project, and had noodled on getting `k8s-infra-gcp-auditor` sufficient privileges to run the audit scripts.